### PR TITLE
increase default timeout for kafka_connect service integration create, add example test

### PR DESCRIPTION
**Pull request by byashimov**
## About this change—what it does

Adds extra timeout for kafka_connect create. Adds example test.

### CODE REVIEW

File: internal/sweep/sweep.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ func SweepServices(region, t string) error {
 			continue
 		}
 
-		if !strings.Contains(s.Name, "test-acc-") {
```
Comment (ivan-savciuc) [inline internal/sweep/sweep.go:58]: Can you also add the a K8s acc and e2e tests prefix?

File: internal/service/service_integration/resource_service_integration.go [hunk 2]
```diff
@@ -148,7 +148,7 @@ func ResourceServiceIntegration() *schema.Resource {
 			StateContext: schema.ImportStatePassthroughContext,
 		},
 		Timeouts: &schema.ResourceTimeout{
-			Create: schema.DefaultTimeout(5 * time.Minute),
+			Create: schema.DefaultTimeout(10 * time.Minute),
```
Comment (ivan-savciuc) [inline internal/service/service_integration/resource_service_integration.go:151]: Customers can configure this field for themselves using something like: 
```
resource "aiven_service_integration" "test" {
  ....
  timeout {
    create = 10m
  }
}
```

**Review by ivan-savciuc — CHANGES_REQUESTED** [review]
Can you change the commit message? 

For example, we claim `fix kafka_connect service integration`, but we have no fix in the code, just a default timeout has been updated, but any user can do that by themselves. What fixed is example, and the current commit message is a bit misleading .

File: internal/service/service_integration/resource_service_integration.go (hunk 2 above)
Comment (byashimov) [reply internal/service/service_integration/resource_service_integration.go:151]: Yeah, I know. But _default_ is not enough.

File: internal/sweep/sweep.go (hunk 1 above)
Comment (byashimov) [reply internal/sweep/sweep.go:58]: Done. But actually operators tests are mess. Naming may vary and doesn't have strict pattern. Can't we just sweep all?

Comment (ivan-savciuc) [reply internal/sweep/sweep.go:58]: I think we can be on the same side if we use `k8s-*` for K8s tests, all of the tests start with this prefix

**Review by ivan-savciuc — DISMISSED** [review]
LGTM

**Comment by ivan-savciuc** [comment]
@byashimov  pls rebase to the latest main, it has some conflicts: 
```
internal/service/service_integration/resource_service_integration.go
```

**Comment by byashimov** [comment]
> @byashimov pls rebase to the latest main, it has some conflicts:
> 
> ```
> internal/service/service_integration/resource_service_integration.go
> ```

Done, thanks.

**Review by Serpentiel — APPROVED** [review]
lgtm

**Pull request merged**